doc/user_guide/Large_Timeseries.ipynb

```diff
@@ -166,22 +173,6 @@
     "This makes LTTB an ideal default method for exploring timeseries datasets, particularly when the dataset size is unknown or too large for standard WebGL rendering."
    ]
   },
-  {
```

Comment:
I'm not really for documenting unreleased features. But something we could do would be to document this as if HoloViews 1.19.0 was already released. In that case, we should also update the code so it's already able to accept and pass the new values. What do you think?

Reply:
Ok, I included a note that these options are available starting in HoloViews 1.19.0. @Hoxbro please be sure to include the tsdownsample [PR](https://github.com/holoviz/holoviews/pull/6059) in the next minor release. I don't think anything further needs to be updated in hvPlot code.


```markdown
### Enhanced Downsampling Options

Starting in HoloViews version 1.19.0, integration with the [tsdownsample](https://github.com/predict-idlab/tsdownsample) library introduces enhanced downsampling functionality with the following methods, which will be accepted as inputs to `downsample` in hvPlot:

- **lttb**: Implements the Largest Triangle Three Buckets ([LTTB](https://github.com/predict-idlab/tsdownsample?tab=readme-ov-file#:~:text=performs%20the-,Largest%20Triangle%20Three%20Buckets,-algorithm)) algorithm, optimizing the selection of points to retain the visual shape of the data.
- **minmax**: For each segment of the data, this method retains the minimum and maximum values, ensuring that peaks and troughs are preserved.
- **minmax-lttb**: A hybrid approach that combines the minmax strategy with LTTB.
- **m4**: A [multi-step process](https://www.vldb.org/pvldb/vol7/p797-jugel.pdf) that leverages the min, max, first, and last values for each time segment.
```

Reply:
>  I don't think anything further needs to be updated in hvPlot code.

Not quite right I think, `downsample` can just be a boolean to use `downsample1d`, I don't think there is a way to customize how it's called from hvPlot at the moment (or if there is, it's not obvious and needs to be documented). I'll open an issue, but since multiple algorithms are documented here, it should be implemented before the release.